client/ayon_core/lib/path_templates.py

```diff
@@ -455,7 +456,7 @@ def validate_value_type(value):
             return True
 
         for inh_class in type(value).mro():
-            if isinstance(inh_class, str):
+            if inh_class is str:
```

Comment:
Do we allow subclasses of `str`? If so, this may need to be `issubclass` instead of an exact type check.

Reply:
Poor me, I should add comment why it was done like this. I don't remember what was the issue exactly but I remember that check of `issubclass` for `str` did not work correctly. I'm not sure if it was Python 2 related or not.

Reply:
A wait, nevermind. You're checking the actual `mro` already. Why, no idea... but ignore my comment then.